frontend/src/tests/setup.js

```diff
@@ -42,6 +42,12 @@ class DatabaseSeeder {
             active_user: {
                 utorid: "smithh",
                 roles: ["admin", "instructor", "ta"]
+            },
```

Comment:
Correct me if I am wrong, but I was under the impression that these seeded data are the minimal data that we need for all the test. For example, because sessions and positions are commonly used in the the whole app, if we don't have these seeded data, we will have to create the data repeatedly and that is why we have them inside setup. But instructor is not a commonly used data for it to be inside seeded data. We should do it the old approach: test apiPOST and then apiGET.